CIP-0013/CIP-0013.md

```diff
@@ -43,27 +55,66 @@ Example:
+
+* any value for the first URI query argument;
+* any URI query argument beyond the first.
+
 ## Security Considerations
 
-1. We cannot prompt the user to send the funds right away as they may not be fully aware of the URI they clicked or were redirected to. Instead, it may be better to simply pre-populate fields in a transaction.
-2. We should be wary of people who disguise “donate” links as actually opening up a phishing website that LOOKS like a wallet.
+1. For payment links, we cannot prompt the user to send the funds right away as they may not be fully aware of the URI they clicked or were redirected to. Instead, it may be better to simply pre-populate fields in a transaction.
+2. For either payment or staking links, we should be wary of people who disguise links as actually opening up a phishing website that LOOKS like that corresponding part of the wallet UI.
 
 # Rationale
 
 ## Why not use Universal links, deep links and other non-protocol-based Solution
 
 An alternative solution to the original problem described above is to use standard URL links in combination with a routing backend system. The routing system is used to redirect to the app's URI. The advantage of this scheme is that it allows to provide a fallback mechanism to handle the case when no application implementing the protocol is installed (for instance, by redirecting to the App Store or Google Play). This is the approach behind iOS Universal Links and Android App Links. In general, it provides a better user experience but requires a centralized system which makes it unsuitable for as a multi-app standard.
 
+## Why stake pool links are important
+
+As K continues to increase, options for convenient and informed re-delegation will be more important as Ada holders face pool saturation more often. Interfaces that connect delegators with pools beyond the highly contested top choices of the in-wallet ranking algorithms are important to avoid saturation and maintain decentralization.
+
+Larger pools and collectives can also use these URIs to link to a family of pools they own to avoid any one of their pools becoming saturated.
+
+Pool links allow for interfaces to initiate delegation transactions without requiring any code modifications to the wallets themselves.
+
+A sudden DeFi migration to Cardano would cause a vast, sudden increase in transaction frequency and depth that the stake pool network will need to support with quality and diversity.  A flow of delegation following small stake pool URIs will help maintain a number of pools sufficient to accommodate Cardano's needs during such expansion periods, by encouraging smaller operators to keep their pools operational and performant.  More meaningful association between a pool's social identity and delegated stake will help ensure that pool operators are sufficiently rewarded by their own communities and other supporters.
```

Comment:
Again, I would avoid talking about things that will happen in the future. These CIPs are support to stay relevant going forward (ex: if somebody reading this in multiple years when DeFi is already popular on Cardano, this section is irrelevant) so I think this paragraph can be removed

Reply:
Removed for re-posting here in the PR thread instead.